README.md

```diff
@@ -3,37 +3,51 @@ Fill in the text below with your own answers to each question. Add your answers
 
 ## About You
 What are your names, as they appear on the class roster?
-> ...
+> Medha Gopalaswamy
+
+> Zoe Wolter
 
 What do you prefer to be called?
-> ...
+> Medha
+
+> Zoe
 
 Why are you taking this course?
-> ...
+> Because I'm a CS major
+
+> To learn about systems
 
 What are your concerns about this course?
-> ...
+> The final project feels slightly intimidating
```

Comment:
The final project should be manageable. You have lots of flexibility in what you work on, but I will do my best to help you pick a reasonable-size project.